torch_xla/csrc/xla_graph_executor.h

```diff
@@ -161,8 +161,8 @@ class XLAGraphExecutor : public torch::lazy::LazyGraphExecutor {
   ComputationCache* GetComputationCache();
 
   std::vector<torch::lazy::BackendDataPtr> ExecuteComputationWithBarrier(
-      torch::lazy::ComputationPtr computation,
-      c10::ArrayRef<torch::lazy::BackendDataPtr> arguments,
+      torch::lazy::hash_t hash,
```

Comment:
Should this be const std::vector<torch::lazy::BackendDataPtr>&?

Reply:
we actually wants to make a copy and pass it to sync fn(separate thread), we can pass a const ref but then we need to make a copy inside the `ExecuteComputationWithBarrier`.

Reply:
Sounds good.